src/openMVG/features/sift/SIFT_vlfeat_describer.hpp

```diff
@@ -0,0 +1,97 @@
+#pragma once
```

Comment:
License! you cannot delete the file in nonfree and copy paste here and removing the license!